web/src/message_notifications.js

```diff
@@ -43,16 +43,12 @@ function get_notification_content(message) {
         content = "New direct message from " + message.sender_full_name;
     }
 
-    if (content.length > 150) {
-        let i;
-        // Truncate content at a word boundary
-        for (i = 150; i > 0; i -= 1) {
-            if (content[i] === " ") {
-                break;
-            }
-        }
-        content = content.slice(0, i);
-        content += " [...]";
+    const characterLimit = 150;
+    const ellipsis = "…";
+
+    if (content.length > characterLimit) {
+        // Truncate content not at a word boundary
+        content = content.slice(0, characterLimit - 3) + ellipsis;
```

Comment:
Several issues with this code:
* Please use `ellipsis.length` rather than 3 -- it's actually only one character now, so your code is wrong.
* `characterLimit` is camel case, which we don't use -- I'd recommend calling it `const NOTIFICATION_CHARACTER_LIMIT = 150` at the top of the file.